src/server/debug/server/server.go

```diff
@@ -401,18 +402,6 @@ func (s *debugServer) collectPachdDumpFunc(pachClient *client.APIClient, limit i
 		if err := s.collectPachdVersion(tw, pachClient, prefix...); err != nil {
 			return err
 		}
-		// Collect the pachd describe output.
```

Comment:
In the case where we filter the debug dump by pachd, it looks like only this function will run, and appLogs() which now collects the describe, logs, lokiLogs won't execute to collect this info.

Reply:
Good catch.  Let me investigate!

Reply:
Fixed.